client/js/app/stores/ExplorerStore.js

```diff
@@ -283,6 +283,10 @@ function _wrapGroupBy(group_by) {
   return group_by;
 }
 
+function _coerceToNumber(str) {
+  return str && typeof(str) === 'string' ? Number(str) : str;
```

Comment:
ok great. I think that makes sense. We can add percentile validation as another piece of work. :)